pkg/agentfs/docker.go

```diff
@@ -115,13 +122,16 @@ func validateEntrypoint(dir string, dockerfileContent []byte, dockerignoreConten
 				return err
 			}
 			if !d.IsDir() && strings.HasSuffix(d.Name(), projectType.FileExt()) {
+				// Exclude files like __init__.py which cannot be entrypoints
+				if d.Name() == "__init__.py" {
```

Comment:
Can probably add `__version__.py` too

Reply:
probably any __ file actually

Reply:
I think `__main__.py` can be a valid entrypoint, in case you also want to use the folder as a module and an executable no? Not an expert.

Reply:
yep good point - i excluded all others